modules/google_service_account/outputs.tf

```diff
@@ -0,0 +1,9 @@
+output "account_email" {
+  description = "The `email` attribute of the `google_service_account` resource."
+  value       = google_service_account.service_account.email
+}
+
+output "service_account_keys_b64" {
+  sensitive = true
```

Comment:
Should this output have a description? Probably better let it error when planning / applying rather than outputting "ERROR". Unless we explain in the description what `ERROR` means or why is legit to have it.

Reply:
Added description